gateway/src/translator.rs

```diff
@@ -494,10 +494,10 @@ impl EthereumTransaction {
     }
 
     /// Retrieve the (localized) Ethereum transaction input.
-    pub fn transaction(&self) -> Fallible<LocalizedTransaction> {
+    pub fn transaction(&self) -> Result<LocalizedTransaction> {
         // Validate method.
         if self.snapshot.input.method != METHOD_TX {
-            return Err(format_err!("not an Ethereum transaction"));
+            return Err(anyhow!("not an Ethereum transaction"));
```

Comment:
`ensure!(self.snapshot.input.method == METHOD_TX, "not an Ethereum transaction");`